app/scripts/controllers/ts/TokenRatesController.js

```diff
@@ -0,0 +1,102 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
This file built would normally be ignored. It was included in the PR so the gulp build pipeline or CI didn't need to be reconfigured for this POC.